flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/FlowReduxStoreBuilder.kt

```diff
@@ -1,20 +1,14 @@
 package com.freeletics.flowredux.dsl
 
-import com.freeletics.flowredux.SideEffect
+import com.freeletics.flowredux.sideeffects.InStateSideEffectBuilder
 import kotlin.reflect.KClass
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 
 @ExperimentalCoroutinesApi
 @FlowReduxDsl
-public class FlowReduxStoreBuilder<S : Any, A : Any> {
+public class FlowReduxStoreBuilder<S : Any, A : Any> internal constructor() {
```

Comment:
This is technically a breaking change but there isn't really a point in anyone using the constructor since they can't do anything with the created instance.